.env

```diff
@@ -15,3 +15,4 @@ SOURCE_REPOSITORY_REF=master
 DOC_LINK ='https://opendatahub.io/docs.html'
 COMMUNITY_LINK ='https://opendatahub.io/community.html'
 ENABLED_APPS_CM = 'odh-enabled-applications-config'
+ALLOW_ADMIN=true
```

Comment:
Is this used anywhere? I see no reference to it in the PR.

Reply:
removed this